plugins/modules/k8s_rollback.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2020, Julien Huon <@julienhuon> Institut National de l'Audiovisuel
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+module: k8s_rollback
+
+short_description: Rollback Kubernetes (K8S) Deployments and DaemonSets
+
+version_added: "2.10"
+
+author:
+    - "Julien Huon (@julienhuon)"
+
+description:
+  - Use the OpenShift Python client to perform the Rollback
+  - Authenticate using either a config file, certificates, password or token.
+  - Similar to the kubectl rollout undo command
+
+options:
+  api_version:
```

Comment:
`community.kubernetes.k8s_name_options` already contains `api_version`, `kind`, `name` and `namespace`